src/Confluent.SchemaRegistry/Rest/DataContracts/Schema.cs

```diff
@@ -0,0 +1,183 @@
+        ///     Returns a string representation of the Schema object.
+        /// </summary>
+        /// <returns>
+        ///     A string that represents the schema object.
+        /// </returns>
+        public override string ToString()
+            => $"{{subject={Subject}, version={Version}, id={Id}, schema={SchemaString}}}";
+        
+        /// <summary>
+        ///     Returns a hash code for this Schema.
+        /// </summary>
+        /// <returns>
+        ///     An integer that specifies a hash value for this Schema.
+        /// </returns>
+        public override int GetHashCode()
+        {
+            int result = Subject.GetHashCode();
+            result = 31 * result + Version;
+            result = 31 * result + Id;
+            result = 31 * result + SchemaString.GetHashCode();
+            return result;
+        }
+        
+        /// <summary>
+        ///     Compares this instance with a specified Schema object and indicates whether this 
+        ///     instance precedes, follows, or appears in the same position in the sort order as
+        ///     the specified schema.
+        /// </summary>
+        /// <param name="other">
+        ///     The schema to compare with this instance.
+        /// </param>
+        /// <returns>
+        ///     A 32-bit signed integer that indicates whether this instance precedes, follows, or
+        ///     appears in the same position in the sort order as the other parameter. Less than 
+        ///     zero: this instance precedes other. Zero: this instance has the same position in
+        ///     the sort order as other. Greater than zero: This instance follows other OR other 
+        ///     is null.
+        /// </returns>
+        public int CompareTo(Schema other)
+        {
+            int result = string.Compare(Subject, other.Subject, StringComparison.Ordinal);
+            if (result == 0)
+            {
+                return Version.CompareTo(other.Version);
+            }
+
+            return result;
+        }
+
+        /// <summary>
+        ///     Determines whether this instance and a specified object, which must also be a Schema 
+        ///     object, have the same value (Overrides Object.Equals(Object))
+        /// </summary>
+        /// <param name="obj">
+        ///     The Schema to compare to this instance.
+        /// </param>
+        /// <returns>
+        ///     true if obj is a Schema and its value is the same as this instance; otherwise, false. 
+        ///     If obj is null, the method returns false.
+        /// </returns>
+        public override bool Equals(object obj)
+        {
+            if (obj == null || GetType() != obj.GetType())
+            {
+                return false;
+            }
+
+            Schema that = (Schema)obj;
+            return Equals(that);
+        }
+
+        /// <summary>
+        ///     Determines whether this instance and another specified Schema object have the same value.
```

Comment:
not clear what "value" is here

Reply:
changed to avoid using the terminology 'value'.